MetriCam2/MetriCam2.csproj

```diff
@@ -98,7 +98,11 @@
       <Link>.licenseheader</Link>
     </None>
     <None Include="packages.config" />
-    <None Include="Resources\MetriCam-Icon.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\DefaultIcon.ico">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
I think CopyToOutputDirectory should be false/never. The icon file is embedded in the generated assembly already, so no need to copy the file itself.
(please test before changing in all cameras)